RecoLocalFastTime/FTLRecProducers/plugins/FTLUncalibratedRecHitProducer.cc

```diff
@@ -71,14 +71,14 @@ FTLUncalibratedRecHitProducer::produce(edm::Event& evt, const edm::EventSetup& e
   evt.getByToken( ftlbDigis_, hBarrel );  
   barrelRechits->reserve(hBarrel->size()/2);
   for(const auto& digi : *hBarrel) {
-    barrelRechits->push_back( std::move(barrel_->makeRecHit(digi)) );
+    barrelRechits->push_back( barrel_->makeRecHit(digi) );
```

Comment:
shouldn't this be changed to emplace_back?

Reply:
OK